site/content/docs/5.0/examples/heroes/index.html

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
need to suctom heroes sample to Orange Brand

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380